src/main/java/com/example/doodle_clone/models/Meeting.java

```diff
@@ -38,6 +38,10 @@ public class Meeting {
 
    private int userID;
 
+   public Meeting(){
+
+   }
+
```

Comment:
I don't know why, but this constructor using lombok gives me an error, so I returned a normal constructor without arguments